packages/contracts-bedrock/scripts/outputs/DeleteOutput.s.sol

```diff
@@ -0,0 +1,314 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { console } from "forge-std/console.sol";
+import { IMulticall3 } from "forge-std/interfaces/IMulticall3.sol";
+
+import { LibSort } from "../libraries/LibSort.sol";
+import { IGnosisSafe, Enum } from "../interfaces/IGnosisSafe.sol";
+import { IL2OutputOracle } from "../interfaces/IL2OutputOracle.sol";
+import { EnhancedScript } from "../universal/EnhancedScript.sol";
+import { GlobalConstants } from "../universal/GlobalConstants.sol";
+
+import { Types } from "../../contracts/libraries/Types.sol";
+import { FeeVault } from "../../contracts/universal/FeeVault.sol";
+import { Predeploys } from "../../contracts/libraries/Predeploys.sol";
+
+/**
+ * @title DeleteOutput
+ * @notice Deletes an output root from the L2OutputOracle.
+ * @notice Example usage is provided in the README documentation.
+ */
+contract DeleteOutput is EnhancedScript, GlobalConstants {
+    /**
+     * @notice A set of contract addresses for the script.
+     */
+    struct ContractSet {
+        address Safe;
+        address ProxyAdmin;
+        address L2OutputOracleProxy;
+    }
+
+    /**
+     * @notice A mapping of chainid to a ContractSet.
+     */
+    mapping(uint256 => ContractSet) internal contracts;
+
+    /**
+     * @notice The Multicall3 contract.
+     * @notice Multicall3 is deployed to a pre-computed address on all supported chains.
+     */
+    IMulticall3 private constant multicall = IMulticall3(0xcA11bde05977b3631167028862bE2a173976CA11);
+
+    /**
+     * @notice The expected versions for the contracts.
+     */
+    string constant internal L2OutputOracle_Version = "1.3.0";
+
+    /**
+     * @notice An array of approvals, used to generate the execution transaction.
+     */
+    address[] internal approvals;
+
+    /**
+     * @notice Place the contract addresses in storage for ux.
+     */
+    function setUp() external {
+        contracts[GOERLI] = ContractSet({
+            Safe: 0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f,
+            ProxyAdmin: 0x01d3670863c3F4b24D7b107900f0b75d4BbC6e0d,
+            L2OutputOracleProxy: 0xE6Dfba0953616Bacab0c9A8ecb3a9BBa77FC15c0
+        });
+    }
+
+    /**
+     * @notice Follow up assertions to ensure that the script ran to completion.
+     */
+    function _postCheck(uint256 index) internal view {
+        IL2OutputOracle l2oo = IL2OutputOracle(contracts[GOERLI].L2OutputOracleProxy);
```

Comment:
We should use `block.chainid` to make this portable instead of hardcoding `GOERLI`

Reply:
Unless we decide on a good solution here i pretty strongly lean on the side of hardcoding so we:
a) don't footgun ourselves in the short-term
b) more easily track in a gardening ticket to make extensible

If you think otherwise, will change it, lmk